packages/lesswrong/components/form-components/ImageUpload.tsx

```diff
@@ -67,15 +68,16 @@ const styles = (theme: ThemeType): JssStyles => ({
 
 const formPreviewSizeByImageType: Record<
   ImageType,
-  {width: number | "auto", height: number}
+  {width: number | "auto", height: number, imgProps?: CloudinaryPropsType}
 > = {
   gridImageId: {
-    width: 203,
-    height: 80
+    width: 250,
+    height: 100
   },
   bannerImageId: {
-    width: "auto",
-    height: 280
+    width: 1600,
+    height: 380,
+    imgProps: {g: 'custom', dpr: '2.0'}
```

Comment:
This makes the banner image preview actually look the same as the sequence banner image on the page (right now it's positioned differently and stretched out)